### PR TITLE
Update to how modes are selected

### PR DESCRIPTION
**Pull request by Tehsurfer**
Switched the dropdown to a title/description format so that we can draw more attention to which mode we are in and give detail about the mode

image

### CODE REVIEW

**Comment by alan-wu** [comment]
@Tehsurfer Please update this pull request with the main branch. The tests should pass now.

File: src/components/FlatmapVuer.vue [hunk 1]
```diff
@@ -2796,6 +2821,13 @@ export default {
   }
 }
 
+.convert-warning-icon-to-info {
```
Comment (alan-wu) [inline src/components/FlatmapVuer.vue:2824]: Is this one used anywhere?

**Review by alan-wu — COMMENTED** [review]
Looks very good, we need to to have a discussion on the description messages.

**Comment by alan-wu** [comment]
By the way, this will not be merged in until the current QA period has passed.

File: src/components/FlatmapVuer.vue (hunk 1 above)
Comment (Tehsurfer) [reply src/components/FlatmapVuer.vue:2824]: oh no will remove it

**Review by alan-wu — APPROVED** [review]

**Pull request merged**